setup.cfg

```diff
@@ -27,15 +27,16 @@ description = Software development kit for open source D-Wave tools
 [options]
 install_requires =
     dimod==0.12.14
-    dwave-cloud-client==0.11.4
+    dwave-cloud-client==0.12.0.dev0    # for next commit: 0.12.0
     dwave-greedy==0.3.0
     dwave-hybrid==0.6.11
-    dwave-inspector==0.5.0
+    dwave-inspector==0.5.0.post0
     dwave-neal==0.6.0
     dwave-networkx==0.8.14
+    dwave-optimization==0.1.0rc1    # for next commit: 0.1.0
     dwave-preprocessing==0.6.5
     dwave-samplers==1.2.0
-    dwave-system==1.24.0
+    dwave-system==1.25.0.dev0    # for next commit: 1.25.0
     dwave-tabu==0.5.0
     dwavebinarycsp==0.3.0
     minorminer==0.2.13
```

Comment:
minorminer will be updated as well

Reply:
Closed by [this commit](https://github.com/dwavesystems/dwave-ocean-sdk/pull/296/commits/fa140f3c4201f03725b7c7846847de946e6e0dd6)